llvm/lib/Target/SystemZ/SystemZAsmPrinter.cpp

```diff
@@ -1014,6 +1016,8 @@ void SystemZAsmPrinter::emitADASection() {
                                 MCSymbolRefExpr::create(Sym, OutContext),
                                 OutContext),
           PointerSize);
+      Sym->setExecutable(GOFF::ESD_EXE_CODE);
+      Sym->setTemporary(false);
```

Comment:
When can this ever be necessary?  It's a function symbol, this should never have been marked as temporary in the first place ...